src/GameObject.h

```diff
@@ -30,6 +30,9 @@ class GameObject {
     // Properties for player moveable objects
     bool toggleMovement;
 
+	// Can a Cookie be delivered to this object
+	bool cookieDeliverable;

```

Comment:
I would name this to something more generic. Recall that we will want to have the Grandpa/Baby throw something other than a cookie.